pages/about.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="" />
+    <meta name="description" content="" />
+
+    <!-- Links -->
+    <link href="../css/about.css" rel="stylesheet" />
+    <link href="../css/sanitize.css" rel="stylesheet" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Tittle -->
+    <title>Capstone Project</title>
+
+    <!-- Font Awesome Icons -->
+    <script
+      src="https://kit.fontawesome.com/2f238780d4.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- Script -->
+    <script src="../index.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <!-- Navigation mobile -->
+
+      <nav class="nav__mobile-menu">
+        <button class="nav__btn-mobile" type="button">
+          <i class="nav__icon-close fas fa-times"></i>
+        </button>
+        <ul class="nav__mobile">
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="../index.html">Home</a>
+          </li>
+          <li class="nav__mobile-list">
+            <a class="nav__mobile-links" href="../pages/about.html">About</a>
+          </li>
```

Comment:
- [ ] After going through your code, I found the cause of the `404 error`. When you are in the about.html which is inside the pages folder, the path to the about page should be `href="about.html"` instead of `href="../pages/about.html"`. This is because you are in the current directory and inside the about.html file which is inside the pages folder. Once you make the below suggested changes, your web pages will not crash again. Please give it a try okay?

```suggestion
          <li class="nav__mobile-list">
            <a class="nav__mobile-links" href="about.html">About</a>
          </li>
```